network-api/networkapi/buyersguide/tests.py

```diff
@@ -329,18 +329,42 @@ def test_localised_homepage(self):
         response = self.client.get('/privacynotincluded/')
         self.assertEqual(response.status_code, 302, 'simple locale gets redirected')
 
+    def test_only_en(self):
+        """
+        Test that the homepage redirects away from the other locales the site supports
+        """
+        response = self.client.get('/fr/privacynotincluded', follow=True)
```

Comment:
Not gonna lie, something weird was going on here. when running the site, I saw 302's in the browser, but when using the test client, it was returning 301. Decided to get the client to follow redirects and check the end result by inspecting the redirect_chain.